API.yaml

```diff
@@ -560,7 +560,7 @@ paths:
             type: string
       requestBody:
         description: Query parameters to fetch the data tree.
-          When 'requested_times' is absent the tree for the full range is returned. When present it specifies a range as [start, end].
+          When 'requested_times' is empty the tree for the full range is returned. When filled it specifies a range as [start, end].
```

Comment:
I'm bit torn. In other places in the TSP (`requested_marker_categories` for fetch annotations) we use absent parameter to specify that all are requested. Maybe we should do it the same way here to be consistent.

Question: Do you think, that we specify what should happens when the array is empty or array is of length 1? For example, empty could mean no values, i.e. filter everything out (like with the requested_marker_categories), and length of one could mean from start to end of trace?

Reply:
> I'm bit torn. In other places in the TSP (`requested_marker_categories` for fetch annotations) we use absent parameter to specify that all are requested. Maybe we should do it the same way here to be consistent.
> 
> Question: Do you think, that we specify what should happens when the array is empty or array is of length 1? For example, empty could mean no values, i.e. filter everything out (like with the requested_marker_categories), and length of one could mean from start to end of trace?

Sure; let's stick to being consistent with other endpoints. I'll remove this commit from the PR. Can you please back-log the gaps though? -Big thanks again.